bin/edit.js

```diff
@@ -15,17 +15,31 @@ module.exports = Cli.createCommand('edit', {
             required: false
         },
     },
+    optionGroups: {
+        'Output options': {
```

Comment:
This should be renamed something a bit more relevant. How about `Editor options`?